### ISSUE TITLE
[Feature Request] Ability to specify day-of-week in schedule

### ISSUE THREAD

**Issue by thx1200** [NeoApplications/Neo-Backup#693]
**Describe the primary solution**
Currently you can specify an interval of days between scheduled backups.  I'd like to be able to specify day-of-week, such as "run this backup every Monday, Wednesday, and Friday."

**Describe alternative solutions, if any**
None that I know of.

**System Information (please complete the following information):**
- Device: Google Pixel 5A
- Android Version: 12
- ROM: Lineage 19.1
- App Version: 8.2.4

**Comment by KoffieNu** [comment]
I agree, being able to set a specific weekday would help a lot. I'd like to have an 'incremental' backup on tuesday and friday and a full one on sunday. Now It it needed to create 3 schedules, 1 for each day, set the interval to 7 days and enable it the desired day.

Device: Samsung Tab S5e (SM-T720), Tab S2 8" (SM-T710), Tab S2ve 10" (SM-T813), Nokia 6.1
Android vesion: 11 (Tab S2s) and 13 (Tab S5e and Nokia 6.1)
Rom: Lineage 18.1 and 20
App version: 8.3.3
